### ISSUE TITLE
Remove unnecessary horizontal scrollbars

### ISSUE THREAD

**Issue by vanjac** [vanjac/chromafiler#23]
In Small Icon view there should be no horizontal scrolling at all.

**Comment by vanjac** [comment]
If List view could be made to scroll vertically it would be an improvement (no unnecessary scrolling, no filename truncation)

**Comment by vanjac** [comment]
IExplorerBrowser window hierarchy:
```
- ExplorerBrowserControl
  - SHELLDLL_DefView
    - DirectUIHWND
      - CtrlNotifySink (may be hidden)
        - ScrollBar
      - CtrlNotifySink (may be hidden)
        - ScrollBar
```

**Comment by vanjac** [comment]
Most important is to prevent auto horizontal scrolling during drag-and-droo.

**Comment by vanjac** [comment]
Fixed with 198ed374161f851a5e38ae0de03447c01736b138
